recipe/meta.yaml

```diff
@@ -22,11 +22,11 @@ source:
     - patches/root-6.18.02-force-search-of-build-related-files.patch
     - patches/root-x.xx.xx_cling-runtime-sysroot.patch
     - patches/root-x.xx.xx_python_mac_os.patch  # [osx]
-    - patches/root-x.xx.xx-copy-abspath-headers.patch  # [osx]
+    # - patches/root-x.xx.xx-copy-abspath-headers.patch  # [osx]
```

Comment:
I think this patch isn't needed any more as its been fixed in https://github.com/root-project/root/commit/31b94463f1b2e21c4a1c56905e6e2d732d587e68